src/main/java/uk/gov/hmcts/ccd/domain/model/definition/CaseType.java

```diff
@@ -153,12 +152,17 @@ public void setSearchAliasFields(List<SearchAliasField> searchAliasFields) {
     }
 
     @JsonIgnore
-    public Optional<CaseField> getCaseField(String caseFieldId) {
-        return caseFields.stream().filter(caseField -> caseField.getId().equalsIgnoreCase(caseFieldId)).findFirst();
+    public CaseField getCaseField(String caseFieldId) {
+        return caseFields.stream().filter(caseField -> caseField.getId().equalsIgnoreCase(caseFieldId)).findFirst()
```

Comment:
can I know what's the rationale for this change? I think the previous solution is cleaner. I don't see why we need to throw an Exception here

Reply:
Changed back